demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -192,6 +192,7 @@ def create_content_repo():
     create_a_pack_entity(common_scripts, FileType.SCRIPT, 'EmailAskUser')
     create_a_pack_entity(common_scripts, FileType.SCRIPT, 'ScheduleCommand')
     create_a_pack_entity(common_scripts, FileType.SCRIPT, 'DeleteContext')
+    create_a_pack_entity(common_scripts, FileType.SCRIPT, 'IsInCidrRanges')
```

Comment:
```suggestion
    create_a_pack_entity(common_scripts, FileType.SCRIPT, 'IsInCidrRanges')
```
@yaakovi do you think we need to add FileType.Transformer and FileType.filter?

Reply:
don't think so, it will cause a lot of overhead since they should be treated as scripts in most places